object_tracker_0/src/describe_annotations.py

```diff
@@ -102,13 +102,17 @@ def log(file: TextIOWrapper, image_id: int, actor: str, message: str):
                     # TODO: give more information of where it appeared, e.g. "appeared in the top left corner" or if it just popped up in the middle of the screen
                     log(output_file, image_id, name, "appeared")
 
+            removed_tracks = []
             for track_id, image_id_last_seen in last_seen.items():
                 if image_id - image_id_last_seen > 32:
-                    name = f"{categories_map[annotation_map[image_id][track_id]['category_id']]}_{track_id}"
```

Comment:
This was a bug, should have been image_id_last_seen, and also you can't alter a collection while iterating